Sidebar style change to match design

pkgamma opened the pull request:
- Updated sidebar styling to match our design mockups
- Made sure MBR percentage color changes based on the value

pkgamma committed:
sidebar style change to match design

pkgamma committed:
sidebar style change to match design

pkgamma commented:
<img width="1680" alt="Screen Shot 2019-11-17 at 7 35 38 PM" src="https://user-images.githubusercontent.com/6351749/69018343-89701580-0971-11ea-9c94-f4a07606d177.png">

"first PR down"
![bitmoji](https://render.bitstrips.com/v2/cpanel/82707dca-5cee-4cf0-93ea-efc0541d7e31-93eef627-21ed-41df-be90-1e8f6d7b3a29-v1.png?transparent=1&palette=1&width=246)

chyku commented on frontend/src/components/Home.js:
```diff
@@ -24,7 +24,7 @@ class Home extends React.Component {
       lat: 38,
       zoom: 3.7,
-      isSidebarOpen: true,
+      isOpen: true,
```
wait, why is this being changed back to `isOpen`? this should be aligned with `master` instead of an old version of megha's branch

pkgamma replied:
Got it.

chyku commented on frontend/src/components/Home.js:
```diff
@@ -33,7 +33,22 @@ class Home extends React.Component {
+    } else if (response_rate <= 100) {
+      return { color: "#7CB342" };
+    }
+  };
```
@caiyingshen did we agree to make ranges of response rates correlate with different colors? i thought it would be more of a gradient but idk how doable that is

pkgamma replied:
This is mainly just a temporary code meant to represent the fact that we will change the text color based on its value. The actual color and separating ranges will probably be changed.

chyku replied:
Please add a comment for that then!

chyku commented on frontend/src/styles/sidebar.css:
```diff
@@ -1,7 +1,13 @@
-  width: 70%;
-  padding-top: 20px;
-  padding-bottom: 20px;
+  width: 162px;
```
do we want this to be more responsive? direct `px` sizes make me a little nervous for how it'll look on different sized screens

pkgamma replied:
I think using direct `px` values should be fine since the sidebar width will be fixed at 375px and the site will not be viewed on a mobile device anyway. (Sidebar width will not be changed dynamically based on the viewport width.) `px` values should work well on tablet devices also, so I think we should be fine.

chyku reviewed (changes requested):
beautiful styling! just need to update the code and a few questions on responsiveness + the colors

pkgamma committed:
changed isOpen to isSidebarOpen

chyku commented on frontend/src/components/Home.js:
```diff
@@ -33,7 +33,22 @@ class Home extends React.Component {
+  };
+
   componentDidMount() {
+    // TODO: Replace the log below with a valid tract request
```
You should also get rid of this line, it's outdated

pkgamma committed:
styled search result autocomplete

pkgamma commented:
- style autocomplete style
- fixed all the above change requests

pkgamma committed:
the good ol npm run format that we love

pkgamma committed:
fixed floating point error

pkgamma commented:
- resolved floating point error where some tracts will show 55.0000000001% instead of 55%
- added shadow to the sidebar, changed search bar style, and etc.

chyku committed:
Merge branch 'master' into sidebar-style-update

Commit:
minor bug fix

chyku committed:
Hover over functionality

chyku committed:
Merge branch 'sidebar-style-update' of https://github.com/hack4impact-uiuc/communityconnect-labs into sidebar-style-update

pkgamma committed:
flipped color values

chyku reviewed (approved):
:fire:

caiyingshen reviewed (approved):
beep beep
